lizmap/www/assets/js/map.js

```diff
@@ -3554,6 +3566,18 @@ var lizMap = function() {
     var unitsRatio = OpenLayers.INCHES_PER_UNIT[units];
     var w = size.width / 72 / unitsRatio * aScale;
     var h = size.height / 72 / unitsRatio * aScale;
+    var printInProjectProjection = true;
+    if ('printInProjectProjection' in lizMap.config.options && lizMap.config.options.printInProjectProjection != 'True') {
+      printInProjectProjection = false;
+    }
+    if (printInProjectProjection
+      && lizMap.config.options.qgisProjectProjection.ref != lizMap.config.options.projection.ref
+    ) {
+      var printDpi = parseFloat(document.querySelector('#print-dpi').value);
```

Comment:
`printDpi` is never used.

Reply:
Does a eslint checks would help here ? to clean step by step the LWC code base ?
Maybe enabling eslint on only modified file in a commit ?

Reply:
Sure, but here it was made on purpose, to let dev who review test different configurations. I removed it after discussion and further tests yesterday by force pushing